hibernate-core/src/main/java/org/hibernate/dialect/SQLServerDialect.java

```diff
@@ -248,4 +251,10 @@ public String getCreateTemporaryTableColumnAnnotation(int sqlTypeCode) {
 				return "";
 		}
 	}
+
+	@Override
+	public NameQualifierSupport getNameQualifierSupport() {
```

Comment:
Interesting that this was missing. Perhaps it deserves its own JIRA so to highlight the change better in changelogs?